Translations/fr.po

```diff
@@ -1126,7 +1126,7 @@ msgstr "Voici quelque chose pour vous."
 
 #: Source/diablo.cpp:123
 msgid "Now you DIE!"
-msgstr "Maintenant tu MEURS!"
+msgstr "MEURS, à présent !"
```

Comment:
I would I loved "meurs, pourriture communiste" but the spirit is too different from the movie :)